contracts/contracts/extensions/CustomFeesFacet.sol

```diff
@@ -2,12 +2,13 @@
 pragma solidity 0.8.18;
 
 import {ICustomFees} from './Interfaces/ICustomFees.sol';
-import {Roles} from './Roles.sol';
-import {TokenOwner} from './TokenOwner.sol';
+import {RolesFacet} from './RolesFacet.sol';
+import {TokenOwnerFacet} from './TokenOwnerFacet.sol';
 // solhint-disable-next-line max-line-length
 import {IHederaTokenService} from '@hashgraph/smart-contracts/contracts/system-contracts/hedera-token-service/IHederaTokenService.sol';
+import {_CUSTOM_FEES_RESOLVER_KEY} from '../constants/resolverKeys.sol';
 
-abstract contract CustomFees is ICustomFees, TokenOwner, Roles {
+contract CustomFeesFacet is ICustomFees, TokenOwnerFacet, RolesFacet {
```

Comment:
This is incorrect. A facet never extends to other Facets:
`contract CustomFeesFacet is ICustomFees, TokenOwnerStorageWrapper, RolesStorageWrapper {`